include/TMS/dev/TMP117.hpp

```diff
@@ -21,7 +21,11 @@ class TMP117 {
      * @param i2c used to read temperature
      * @param deviceID - device address
      * */
-    TMP117(IO::I2C& i2c, uint8_t deviceID);
+    TMP117(IO::I2C* i2c, uint8_t deviceID);
+    /**
+     * second constructor to make TMS main work
```

Comment:
Add a newline between the previous constructor and the comment. This comment should also explain that this is a default constructor for instantiation of uninitialized instances instead of just saying "to make main work"